motor/core.pyi

```diff
@@ -572,7 +568,7 @@ class AgnosticCollection(AgnosticBaseProperties[_DocumentType]):
         collation: Optional[_CollationIn] = None,
         array_filters: Optional[Sequence[Mapping[str, Any]]] = None,
         hint: Optional[_IndexKeyHint] = None,
-        session: Union[Optional[AgnosticClientSession], Optional[AgnosticClientSession]] = None,
```

Comment:
Interesting. Why was this ever needed?

Reply:
It wasn't, I think it is a copy-paste artifact.